layer.md

```diff
@@ -156,3 +156,13 @@ Note that this opaque file will apply to _all_ children, including sub-directori
 Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
 
 Any given image is likely to be composed of several of these Image Filesystem Changeset tar archives.
+
+# Foreign Layers
+
+Certain layers, due to legal requirements, may not be regularly distributable.
+Typically, one can download such layers but they should never be uploaded.
+
+Layers that have these restrictions SHOULD be tagged with an alternative mediatype of `application/vnd.oci.image.layer.foreign.tar+gzip`.
+[Descriptors](descriptor.md) referencing these layers MAY include `urls` for downloading these layers.
+It is implementation defined whether or not implementations upload layers tagged with this media type.
```

Comment:
@runcom I think there were concerns about the meaning of distribute. I'll change it to that if we can re-obtain consensus.

@stephenrwalli CC